### ISSUE TITLE
Setup a new deployment and teardown for RC testing of runtime image

### ISSUE THREAD

**Issue by pankajkoti** [astronomer/astronomer-providers#851]
Currently, we use the existing integration tests deployments for testing RC runtime image.
Add a make target to create setup new deployment and tear it down for testing RCs for runtime images.

**Comment by pankajastro** [comment]
How do we set up connections? otherwise, it would be time taking process on new deployment. Also, much us do not have access to create connections.

**Comment by phanikumv** [comment]
I dont think this is a possibility considering we cant copy connection information
